fix: Update macos.mdx (#6376)
FirebaseExtended/flutterfire: docs/installation/macos.mdx

    <true/>
</dict>
```

> ## Enabling use of Firebase Emulator Suit for Auth (MacOS only)

Reviewer: ```suggestion
## Enabling use of Firebase Emulator Suit for Auth
```

> For mac os you need to **allow Outgoing Connections on the Xcode's Runner.xcworkspace** 

Reviewer: ```suggestion
MacOS development requires you enable `Outgoing Connections (Client)` on Xcode's Runner.xcworkspace. 
```

> ![image](https://user-images.githubusercontent.com/12681781/121333660-d1330500-c93a-11eb-8fb6-b1d803b9c7bb.png)

Reviewer: ```suggestion
![image](https://user-images.githubusercontent.com/12681781/121333660-d1330500-c93a-11eb-8fb6-b1d803b9c7bb.png).
```

> here is the full [thread ](https://stackoverflow.com/a/66008698/6133329)

Reviewer: ```suggestion
Here is a related StackOverflow [thread](https://stackoverflow.com/a/66008698/6133329).
```

> otherwise you might get

Reviewer: ```suggestion
Otherwise you might get the following error:
```
